CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+# Contributing
+1. Fork this repository to your own account
+2. Make your changes and verify that tests pass
+3. Commit your work and push to a new branch on your fork
+4. Submit a pull request
+5. Participate in the code review process by responding to feedback
+
+
+### Mac M1 contributors
```

Comment:
please add an instruction that you need to install the image of mysql:
``

Reply:
There is no need in it because if we set `platform` flag in docker configuration, everything works fine on M1. Need to check on other platforms.